bin/ci/run-tests.js

```diff
@@ -58,7 +58,21 @@ Object.keys(fileSizes).forEach(key => {
 });
 
 const otherTests = [
-  "browser_net_open_in_debugger.js"
+  "devtools/client/framework/test/browser_browser_toolbox_debugger.js",
+  "devtools/client/framework/test/browser_keybindings_01.js",
+  "devtools/client/framework/test/browser_source_map-01.js",
+  "devtools/client/framework/test/browser_source_map-absolute.js",
+  "devtools/client/framework/test/browser_source_map-init.js",
+  "devtools/client/framework/test/browser_source_map-inline.js",
+  "devtools/client/framework/test/browser_source_map-reload.js",
+  "devtools/client/framework/test/browser_toolbox_getpanelwhenready.js"
+  "devtools/client/netmonitor/test/browser_net_open_in_debugger.js",
+  "devtools/client/netmonitor/test/browser_net_view-source-debugger.js",
+  "devtools/client/webconsole/new-console-output/test/mochitest/browser_jsterm_autocomplete_in_debugger_stackframe.js",
+  "devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_eval_in_debugger_stackframe1.js",
+  "devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_eval_in_debugger_stackframe2.js",
+  "devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_location_debugger_link.js",
+  "devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_stacktrace_location_debugger_link.js",
```

Comment:
any inspector tests?

Reply:
Ah yes there seems to at least be one: devtools/client/inspector/markup/test/browser_markup_links_06.js not sure how I missed it my first pass. Thanks!